.github/workflows/license-check.yml

```
@@ -0,0 +1,40 @@
+name: Check licenses
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  license_check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Node
+        uses: actions/setup-node@v1
+        with:
+          node-version: '16.14.2'
+      - name: Checkout Ref
+        uses: actions/checkout@v1
```

Nitpick: I find the workflow files easier to read if there is a newline between actions.
```suggestion

      - name: Checkout Ref
        uses: actions/checkout@v1

```

---

done